isis/tests/FunctionalTestsNoCam2Map.cpp

```diff
@@ -0,0 +1,132 @@
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "LineManager.h"
+#include "Histogram.h"
+
+#include "nocam2map.h"
+
+#include "gtest/gtest.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/nocam2map.xml").expanded();
+
+TEST_F(MroHiriseCube, FunctionalTestsNoCam2MapDefault) {
+  
+  QString outCubeFileName = tempDir.path() + "/outTEMP.cub";
+  QString lonFile = tempDir.path() + "/lons.cub"; 
+  QString latFile = tempDir.path() + "/lats.cub"; 
+  
+  Cube *latCube = new Cube();
+  latCube->setDimensions(testCube->sampleCount(), testCube->lineCount(), 1);
+  latCube->create(latFile);
+
+  LineManager latline(*latCube);
+  double latValue = -60;
+  for(latline.begin(); !latline.end(); latline++) {
+    for(int i = 0; i < latline.size(); i++) {
+      latValue += 0.0001;
+      latline[i] = (double) latValue ;
+    }
+    latCube->write(latline);
+  }
+  latCube->close(); 
```

Comment:
This is redundant between the two tests, can't really justify not factoring it out as I cannot imagine other tests need Lat/Lon cubes. I'll double check in the AM and if there is a good reason to make it a fixture I can,

Reply:
Could make it into a fixture local to these tests, but I'm fine with it either way